### PR TITLE
Move remaining metadata to pyproject.toml

### PR DESCRIPTION
**Pull request by cdce8p**
## Description

The metadata changes:

```diff
 Metadata-Version: 2.1
 Name: pywebpush
 Version: 2.0.1
 Summary: WebPush publication library
-Author: JR Conlin
 Author-email: JR Conlin <src+webpusher@jrconlin.com>
-License: MPL2
+License: MPL-2.0
-Home-page: https://github.com/web-push-libs/pywebpush
+Project-URL: Homepage, https://github.com/web-push-libs/pywebpush
 Keywords: webpush,vapid,notification
 Classifier: Topic :: Internet :: WWW/HTTP
 Classifier: Programming Language :: Python :: Implementation :: PyPy
 Classifier: Programming Language :: Python
 Classifier: Programming Language :: Python :: 3
 Classifier: License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)
 Description-Content-Type: text/markdown
 License-File: LICENSE
 Requires-Dist: aiohttp
-Requires-Dist: cryptography>=2.6.1
+Requires-Dist: cryptography >=2.6.1
-Requires-Dist: http-ece>=1.1.0
+Requires-Dist: http-ece >=1.1.0
-Requires-Dist: requests>=2.21.0
+Requires-Dist: requests >=2.21.0
-Requires-Dist: six>=1.15.0
+Requires-Dist: six >=1.15.0
-Requires-Dist: py-vapid>=1.7.0
+Requires-Dist: py-vapid >=1.7.0
 Provides-Extra: dev
-Requires-Dist: black; extra == "dev"
+Requires-Dist: black ; extra == 'dev'
-Requires-Dist: mock; extra == "dev"
+Requires-Dist: mock ; extra == 'dev'
-Requires-Dist: pytest; extra == "dev"
+Requires-Dist: pytest ; extra == 'dev'

 ...
```

- **License**: Use SPDX license identifier
- Remove `wheel` from `build-system.requires`. Recent setuptool versions bundle `bdist_wheel`, so it's no longer necessary
- `tests_require`: The setuptools test command was removed

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
 # details that you may not care about.)
-requires = ["setuptools", "wheel"]
+requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
 [project]
-# `dependencies` are taken from `setup.py` and include the contents of the
-# `requirements.txt` file
 name = "pywebpush"
+version = "2.0.1"
```
Comment (jrconlin) [inline pyproject.toml:12]: Need to bump this one more than the latest. (so it should be "2.0.3")

**Review by jrconlin — CHANGES_REQUESTED** [review]
Also need to merge against master since there are some conflicts.

Otherwise, looks good.

**Comment by cdce8p** [comment]
Done. Saw that you removed `license="MPL2"` in https://github.com/web-push-libs/pywebpush/pull/172. Note that this PR adds
```
license = {text = "MPL-2.0"}
```
to pyproject which maps to the same field. I believe it's fine as `MPL-2.0` is the official SPDX identifier and thus doesn't conflict with the license classifier.

At some point this could even be changed to just `license = "MPL-2.0"` once setuptools adds support for PEP 639.

**Review by jrconlin — APPROVED** [review]

**Comment by jrconlin** [comment]
Sigh... 
And github just noted that the commit was unsigned. 
I'll force it through for now, but in the future, please sign commits. Thanks!

**Comment by cdce8p** [comment]
> Sigh... And github just noted that the commit was unsigned. I'll force it through for now, but in the future, please sign commits. Thanks!

Sorry about that.

If you've got some time. I opened a similar PR for `vapid`: https://github.com/web-push-libs/vapid/pull/106

**Pull request merged**